stripe2qbo/db/models.py

```diff
@@ -28,6 +29,8 @@ class User(Base):
     stripe_user_id: Mapped[str | None] = mapped_column(nullable=True)
     qbo_token: Mapped["QBOToken"] = relationship("QBOToken", uselist=False)
 
+    transactions = relationship("TransactionSync", back_populates="id")
```

Comment:
`back_populates="user"` (the name of the relationship field on `TransactionSync`)